wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -545,11 +552,21 @@ public static void updateBeanValue(final WComponent component)
             // Update the rest of the bean
             for (int i = ((Container) component).getChildCount() - 1; i >= 0; i--)
             {
-                updateBeanValue(((Container) component).getChildAt(i));
+                updateBeanValue(((Container) component).getChildAt(i), visibleOnly);
             }
         }
     }
-
+    
+        /**
+     * Updates the bean value with the current value of the component
+     * and all its bean-bound children. By default this method will process invisible containers and inputs.
+     *
+     * @param component the component whose contents need to be copied to the bean.
+     */
+    public static void updateBeanValue(final WComponent component)
+    {
+            updateBeanValue(component, false);

```

Comment:
Did you mean this to be true? Did we want to make "true" the new default behaviour so only "visible" is processed? We will then have to explain this in the change log.